src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
its forbidden to use anything except box-sizing in `* {...}`